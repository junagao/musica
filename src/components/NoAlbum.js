import React from 'react';

import './NoAlbum.scss';

const NoAlbum = () => <div className="no-album">Sorry, no album found.</div>;

export default NoAlbum;
