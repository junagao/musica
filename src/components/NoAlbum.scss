.no-album {
  font-size: 1.25rem;
  padding: 2rem 0rem;
}
