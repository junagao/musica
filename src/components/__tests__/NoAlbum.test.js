import React from 'react';
import { shallow } from 'enzyme';

import NoAlbum from 'components/NoAlbum';

describe('NoAlbum', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NoAlbum />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render one div element with className of no-album', () => {
    expect(wrapper.find('div').length).toEqual(1);
    expect(wrapper.find('div').hasClass('no-album')).toEqual(true);
  });

  it('should render the error message', () => {
    expect(wrapper.find('div').text()).toEqual('Sorry, no album found.');
  });
});
