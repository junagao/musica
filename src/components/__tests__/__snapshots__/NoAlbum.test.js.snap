// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoAlbum should render correctly 1`] = `
<div
  className="no-album"
>
  Sorry, no album found.
</div>
`;
